### DISCUSSION TITLE
High CPU usage with coral devices -- is this normal?

### DISCUSSION THREAD

**Discussion by brettks** [blakeblackshear/frigate#13713] [Ask A Question] [answered]
Brief background, I'm running frigate within docker on an ubuntu host. I have two coral devices, one pcie and one usb. They have worked well with my config for some time. I do not think I have changed anything, but I've noticed that CPU usage is quite high. (Started investigating this when my server had some random crashes/unresponsiveness and trying to figure out what's causing it)

I wonder if an issue related to frigate CPU usage is the culprit.
System stats here:
![image](https://github.com/user-attachments/assets/ae342879-d349-4425-99e7-bf63e7aedb55)

top output:
![image](https://github.com/user-attachments/assets/cddbdfd0-15ec-43dd-91be-5bfd588bd953)

Config here:
[config.yml.txt](https://github.com/user-attachments/files/16985838/config.yml.txt)

**Comment by hawkeye217** [comment, marked as answer]
You should disable RTMP (`enabled: False`) and set up [hardware acceleration](https://docs.frigate.video/configuration/hardware_acceleration/).

**Reply by brettks** [reply]
That fixed it. Appreciate the tips!

**Reply by hawkeye217** [reply]
Great! Happy to help.
